contracts/VotingProxy.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import "./factories/BallotFactory.sol";
+
+contract VotingProxy is Ownable {
+    IERC20Metadata public immutable DPS;
+    BallotFactory public ballotFactory;
+    BallotTagManager public ballotTagManager;
+
+    struct Grants {
+        mapping(address => uint32) voterIndex;
+        mapping(uint32 => address) indexVoter;
+        uint32 grantCount;
+    }
```

Comment:
Can be replaced with:
```suggestion
    struct Grants {
        mapping(address => uint32) indices;
        address[] delegators;
    }
```

`delegators.length` is the equivalent of `grantCount`.